xicam/gui/bluesky/search.py

```diff
@@ -407,24 +408,80 @@ def __init__(self, *args, **kwargs):
         self.horizontalHeader().setSectionResizeMode(QHeaderView.ResizeToContents)
         self.setAlternatingRowColors(True)
 
+    def hide_hidden_columns(self):
+        hidden_columns = QSettings().value("catalog.columns.hidden")
+        if hidden_columns is None:
```

Comment:
Can we initialize `hidden_columns` as an empty set to avoid needing to perform this special-case check here and in places later on?

Reply:
:+1: